docs/zh/docs/kpanda/user-guide/workloads/create-deployment.md

```diff
@@ -105,7 +105,7 @@
 
 === "数据存储（选填）"
 
-    配置容器挂载数据卷和数据持久化的设置。详情可参考[容器数据存储配置](pod-config/env-variables.md)。
+    配置容器挂载数据卷和数据持久化的设置。详情可参考[容器数据存储配置](storage/pv.md)。
```

Comment:
This link does not work. See preview:
https://deploy-preview-6389--daocloud-docs.netlify.app/kpanda/user-guide/workloads/create-deployment#__tabbed_1_5